### PR TITLE
HDDS-4416. Update README.md after TLP separation

### PR DESCRIPTION
**Pull request by elek**
## What changes were proposed in this pull request?

Update README.md with the TLP information

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4416

## How was this patch tested?

Text only change. :eyes:

### CODE REVIEW

**Review by linyiqun — APPROVED** [review]
+1. Thanks for updating this.

File: README.md [hunk 1]
```diff
@@ -53,7 +51,7 @@ aws s3 --endpoint http://localhost:9878 cp --storage-class REDUCED_REDUNDANCY  /
 
 ### Run Ozone from released artifact
 
-If you need a more realistic cluster, you can [download](https://hadoop.apache.org/ozone/downloads/) the last (binary) release package, and start a cluster with the help of docker-compose:
+If you need a more realistic cluster, you can [download](https://ozone.apache.org/downloads/) the last (binary) release package, and start a cluster with the help of docker-compose:
```
Comment (adoroszlai) [inline README.md:54]:
```suggestion
If you need a more realistic cluster, you can [download](https://ozone.apache.org/downloads/) the latest (binary) release package, and start a cluster with the help of docker-compose:
```

Not introduced in this patch, but it caught my eye.

**Review by adoroszlai — APPROVED** [review]
Thanks @elek for taking care of this.

File: README.md (hunk 1 above)
Comment (elek) [reply README.md]: > Not introduced in this patch...

Not a problem and thanks the fix. I :heart: the boy-scout rule...

(OFF: which is also mentioned in this article what I read today: https://blog.rstankov.com/collaborative-single-player-mode)

**Review by dineshchitlangia — APPROVED** [review]
+1 Thanks @elek  for the changes and others for the review.

**Comment by cxorm** [comment]
Hi @elek, thank you for updating this after TLP separation.

Would we change the line 80  
`cd hadoop-ozone/dist/target/ozone-*/compose/ozone`
to
`cd ozone/dist/target/ozone-*/compose/ozone` ?

I think it is my nit, but, if yes
I propose we could also update the link and add new label on Github About Part : )
Github_AboutPart

**Comment by elek** [comment]
Good findings @cxorm. Let me create a follow-up PR.

**Comment by elek** [comment]
> Would we change the line 80 `cd hadoop-ozone/dist/target/ozone-*/compose/ozone` to `cd ozone/dist/target/ozone-*/compose/ozone ?`

Project names are not (yet) changed. It's a bigger task:

 1. we can rename those `hadoop-` prefixes from the dir
 2. we can change package names
 3. we can update pom.xml descriptions.

But these are bigger tasks (imho)...

**Comment by cxorm** [comment]
> Good findings @cxorm. Let me create a follow-up PR.

Thank you @elek for the task : )

> Project names are not (yet) changed. It's a bigger task:
> 
> 1. we can rename those `hadoop-` prefixes from the dir
> 2. we can change package names
> 3. we can update pom.xml descriptions.
> 
> But these are bigger tasks (imho)...

Yeah, thanks for the clarification, I also agree with it.

**Pull request merged**